.github/workflows/build_with_current_nightly.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       - name: update nightly toolchain
         run: |
           rustup update nightly
-          rustup default nightly
           rustup component add rust-src --toolchain nightly
-          cargo ledger setup
+          export RUST_NIGHTLY=nightly
```

Comment:
Isn't this necessary at build step too?
I'm afraid it's not persisted between steps?